app/scripts/controllers/swaps.js

```diff
@@ -101,19 +110,35 @@ export default class SwapsController {
     this._setupSwapsLivenessFetching()
   }
 
+  // Sets the refresh rate for quote updates from the MetaSwap API
+  async _setSwapsQuoteRefreshTime() {
+    let refreshTime
+    try {
+      refreshTime = await this._fetchSwapsQuoteRefreshTime()
+    } catch (e) {
+      refreshTime = FALLBACK_QUOTE_REFRESH_TIME
+      console.error('Request for swaps quote refresh time failed: ', e)
```

Comment:
It would be nice to collect metrics on these failures :thinking:
Not blocking for this PR, but if this did fail frequently in production, we might not have any insight into that. Especially if the server response was successful.